src/python/WMCore/WebTools/Root.py

```diff
@@ -93,16 +93,17 @@ def access(self):
         if not rbytes:
             try:
                 # request.rfile.rfile.bytes_read is a custom CMS web
-                #  cherrypy patch not always available, hence the test
-                rbytes = (getattr(request.rfile, 'rfile', None)
-                        and getattr(request.rfile.rfile, "bytes_read", None)
-                        and request.rfile.rfile.bytes_read) or "-"
+                # cherrypy patch not always available, hence the test
+                if request.rfile.rfile.bytes_read:
```

Comment:
This is not safe as it will throw exception if any nested member data does not exist. You cannot use if for that. Instead, you should walk through every structure and identify if there is specific data you request in it, see my suggestion below.

Reply:
It is fine with me, I just pointed out to potential problem. Anyway, I still suggest to add proper unit test to test all possible scenarios how to access rbytes value and check that no identation error will be thrown.